vphysics_jolt/cbase.h

```diff
@@ -41,7 +41,7 @@
 #include "tier1/tier1.h"
 #include "tier1/strtools.h"
 #include "tier1/interface.h"
-#include "tier1/keyvalues.h"
+#include "tier1/KeyValues.h"
```

Comment:
Hm, in our code base for Desolation this is lowercase. Ugh.

Reply:
Can we put this behind the `#ifndef GAME_L4D2_OR_NEWER` that I just added to main?
Looks like that's the first branch where this casing changed.

Reply:
Actually I will just do this myself to spare you the pain of rebasing.